release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -447,6 +447,18 @@ We will have details here when {product-title} {product-version} is released.
 ==== Support for managing the Gateway API custom resource definition (CRD) lifecycle
 We will have details here when {product-title} {product-version} is released.
 
+[id="ocp-4-19-networking-ptp-dual-oc_{context}"]
+==== Dual-port NICs for improved redundancy in PTP ordinary clocks (Technology Preview)
```

Comment:
```suggestion
==== Dual-port network interface controllers (NICs) for improved redundancy in PTP ordinary clocks (Technology Preview)
```

Reply:
Added the definition of NIC to the first line to avoid a long title